vscode/microsoft-kiota/package.json

```diff
@@ -3,8 +3,8 @@
   "displayName": "Microsoft Kiota",
   "publisher": "ms-graph",
   "description": "Client generator for HTTP REST APIs described by OpenAPI which helps eliminate the need to take a dependency on a different API client for every API that you need to call, as well as limiting the generation to the exact API surface area you're interested in, thanks to a filtering capability.",
-  "version": "1.12.100000001",
-  "kiotaVersion": "1.12.0",
+  "version": "1.18.100000001",
+  "kiotaVersion": "1.17.0",
```

Comment:
Should the plugin and kiota version be the same here?

Reply:
I don't believe this should show up in the diff of this PR (a rebase is probably needed). 

For context, the versions are like this due to local testing/debugging (if you point to a version that available in production it will make the extension download an executable version that does not have the changes you want) but the release infrastructure does automatically set this correctly on deployment. 

https://github.com/microsoft/kiota/blob/main/vscode/microsoft-kiota/debugging.md